internal/manifest/os.go

```diff
@@ -465,12 +465,16 @@ func (p *OS) serialize() osbuild.Pipeline {
 	}
 
 	if p.Subscription != nil {
-		commands := []string{
-			fmt.Sprintf("/usr/sbin/subscription-manager register --org=%s --activationkey=%s --serverurl %s --baseurl %s", p.Subscription.Organization, p.Subscription.ActivationKey, p.Subscription.ServerUrl, p.Subscription.BaseUrl),
```

Comment:
afaik we always need subscription-manager as a base.

Reply:
The way I read the documentation is that it replaces it (or uses it behind the scenes). The rhc connect arguments are pretty similar, except for not having a baseurl. But without actually being able to run this I'm not 100% sure :)

Reply:
ah I just tried it out, it'll just do the subman + insights steps for you. So you're right.